Modules/Quotas/src/Quotas.Domain/DomainErrors.cs

```diff
@@ -13,4 +13,9 @@ public static DomainError MaxValueCannotBeLowerOrEqualToZero()
     {
         return new DomainError("error.platform.quotas.invalidValueForMaxLimitInQuota", "A quota max value cannot be lower of equal to zero.");
     }
+
+    public static DomainError DuplicateQuota()
+    {
+        return new DomainError("error.platform.quotas.duplicateQuota", "There is already a quota for this Tier/Identity targeting the same Metric and the same Period.");
```

Comment:
I would remove the "Tier/Identity" part, because it may be confusing.
```suggestion
        return new DomainError("error.platform.quotas.duplicateQuota", "A quota targeting the same metric and period already exists.");
```